mobile/src/main/java/org/openhab/habdroid/background/ItemsControlsProviderService.kt

```diff
@@ -97,7 +97,13 @@ class ItemsControlsProviderService : ControlsProviderService() {
                     }
                     val topic = event.getString("topic")
                     val topicPath = topic.split('/')
-                    if (topicPath.size != 4) {
+                    /* Possible formats:
+                    * - openhab/items/<item>/statechanged
```

Comment:
The asterisk alignment bothers my OCD ;-) Maybe use normal comments instead of block comment?